server/athenian/api/experiments/preloading/entries.py

```diff
@@ -109,6 +109,113 @@ class PreloadedReleaseToPullRequestMapper(ReleaseToPullRequestMapper):
     release_loader = PreloadedReleaseLoader
 
 
+class PreloadedMergedPRFactsLoader(MergedPRFactsLoader):
+    """Loader for preloaded merged PRs facts."""
+
+    @classmethod
+    @sentry_span
+    async def load_merged_unreleased_pull_request_facts(
+            cls,
+            prs: pd.DataFrame,
+            time_to: datetime,
+            labels: LabelFilter,
+            matched_bys: Dict[str, ReleaseMatch],
+            default_branches: Dict[str, str],
+            release_settings: ReleaseSettings,
+            account: int,
+            pdb: databases.Database,
+            time_from: Optional[datetime] = None,
+            exclude_inactive: bool = False,
+    ) -> Dict[str, PullRequestFacts]:
+        """
+        Load the mapping from PR node identifiers which we are sure are not released in one of \
+        `releases` to the serialized facts.
+
+        For each merged PR we maintain the set of releases that do include that PR.
+
+        :return: Map from PR node IDs to their facts.
+        """
+        if time_to != time_to:
+            return {}
+        assert time_to.tzinfo is not None
+        if exclude_inactive:
+            assert time_from is not None
+        # log = logging.getLogger("%s.load_merged_unreleased_pull_request_facts" %
+        #                         metadata.__package__)
+        # ghmprf = GitHubMergedPullRequestFacts
+        # postgres = pdb.url.dialect in ("postgres", "postgresql")
+        # selected = [ghmprf.pr_node_id,
+        #             ghmprf.repository_full_name,
+        #             ghmprf.data,
+        #             ghmprf.author,
+        #             ghmprf.merger,
```

Comment:
This looks suspicious :smile:

Reply:
HAHAHAHAHA damn 😂
I got confused with the different branches and forgot that this one was not ready 😂😂😂